RELEASENOTES/v2.0.0.alpha-3.MD

```diff
@@ -0,0 +1,59 @@
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-alpha.3)](https://github.com/notaryproject/notation/releases/tag/v0.79.0-alpha.1)***   (Updated in this release) 
+## Goal of the release
+- Update Notary v2 specifications 
+    - Community can use the updated specifications to develop their own implementation of Notary v2 in language/platforms of their choice, and/or contribute to Notation client development.
+    - Signature format (JWS) for RC-1.
+    - Updates on use of plugins for signing and verification. Plugins let vendors and users integrate base Notation client with their choice of Key Vaults and PKI.
+ - Update "Notation" (CLI client) implementation to match the latest specifications. With the new client, users can
+    - Sign images with remote key stores that securely store the signing keys
+    - Verify signatures using Trust stores configured in Notation clients
+    - Sign images and Verify signatures with locally stored test keys/certificates for demonstration use only
+    - Setup Trust stores with the new directory-based structure
+    - Configure Trust Policy as a JSON document. Support for registry scope and signature verfication levels to customize the behaviour during verification.
+    - Store signatures in registries compliant to the ORAS Reference spec [RC-1](https://github.com/oras-project/oras-go/releases/tag/v2.0.0-rc.2)
+
+*We do not recommend this release for use in production environment. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notary v2 specifications
+Updated Trust Policy, Trust Store,  Signature Verification and Signing workflows along with extensibility mechanisms via Notation plugins.
+- A final JWS based [signature envelope format.](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md)
+- Updated [Signature specfication](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md) with extensibility provisions for additional signature envelope format.
+- Updated [signing and verification workflows](https://github.com/notaryproject/notaryproject/blob/main/signing-and-verification-workflow.md)
+- Introducing a new [Signing scheme](https://github.com/notaryproject/notaryproject/blob/main/signing-scheme.md) to integrate Notation Client with Signing Services
+- Updated  [trust store and trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md) specification
```

Comment:
IMO - we should make sure we have documentation supporting how to use these manually.

Reply:
@dtzar - Agreed. We can add a link to as part of a separate PR.